README.md

```diff
@@ -361,6 +361,7 @@ If you want some CLI options to stick around, you can configure them in your
 ```vim
 let test#ruby#minitest#options = '--verbose'
 let test#javascript#denotest#options = '--quiet'
+let test#go#gotest#args = 'a=b'
```

Comment:
Instead of adding it here, could you add some documentation like the following around line 400


> The gotest runner let you specify the `-args` argument as follows:
> 
> ```vim
> let test#go#gotest#args = 'a=b'
> ```

Reply:
ok done. let me know if is ok.